tests/unit/s2n_tls13_record_aead_test.c

```diff
@@ -381,7 +381,7 @@ int main(int argc, char **argv)
 
             EXPECT_SUCCESS(s2n_stuffer_wipe(&conn->out));
 
-            EXPECT_SUCCESS(s2n_record_write(conn, APPLICATION_DATA, &in));
```

Comment:
It looks like this value is always overwritten with TLS_APPLICATION_DATA when writing a tls13_record (https://github.com/awslabs/s2n/blob/master/tls/s2n_record_write.c#L232). Am I following that correctly? Should this test purposefully write an incorrect content_type to verify that the content_type is replaced with TLS_APPLICATION_DATA?

Reply:
you're following that correctly, I can change this to another TLS type to make it clearer